File: undefined/tasks/create-task/code-samples/createTask.php
<?php
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
$chatId = '<ENTER VALUE>';

// POST BODY
$body = array(
    'subject' => '<ENTER VALUE>',
    'assignees' => array(
        array(
            'id' => '<ENTER VALUE>'
        )  
    ),
    'completenessCondition' => 'Simple',
    'startDate' => '<ENTER VALUE>',
    'dueDate' => '<ENTER VALUE>',
    'color' => 'Black',
    'section' => '<ENTER VALUE>',
    'description' => '<ENTER VALUE>',
    'recurrence' => array(
        'schedule' => 'None',
        'endingCondition' => 'None',
        'endingAfter' => 000,
        'endingOn' => '<ENTER VALUE>'
    ),
    'attachments' => array(
        array(
            'id' => '<ENTER VALUE>'
        )  
    )
)

require('vendor/autoload.php');
$rcsdk = new RingCentral\SDK\SDK(getenv('clientId'), getenv('clientSecret'), getenv('serverURL'));
$platform = $rcsdk->platform();
$platform->login(getenv('username'), getenv('extension'), getenv('password'));
$r = $platform->post("/restapi/v1.0/glip/chats/{$chatId}/tasks", $body);
?>

File: undefined/tasks/create-task/code-samples/createTask.cs
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
string chatId = "<ENTER VALUE>";

// POST BODY
GlipCreateTask glipCreateTask = new GlipCreateTask {
    subject = "<ENTER VALUE>",
    assignees = new[] {
        new  {
            id = "<ENTER VALUE>"
        },
    },
    completenessCondition = 'Simple',
    startDate = "<ENTER VALUE>",
    dueDate = "<ENTER VALUE>",
    color = 'Black',
    section = "<ENTER VALUE>",
    description = "<ENTER VALUE>",
    recurrence = new  {
        schedule = 'None',
        endingCondition = 'None',
        endingAfter = 000,
        endingOn = "<ENTER VALUE>"
    },
    attachments = new[] {
        new  {
            id = "<ENTER VALUE>"
        },
    }
};

RestClient rc = new RestClient(
    Environment.GetEnvironmentVariable("clientId"),
    Environment.GetEnvironmentVariable("clientSecret"),
    false
);
await rc.Authorize(
    Environment.GetEnvironmentVariable("username"),
    Environment.GetEnvironmentVariable("extension"),
    Environment.GetEnvironmentVariable("password")
);
var r = await rc.Restapi().Glip().Chats(chatId).Tasks().Post(glipCreateTask);
// PROCESS RESPONSE
